include/yacx/Exception.hpp

```diff
@@ -114,6 +114,21 @@ inline void __checkNvrtcResultError(const nvrtcResult error, const char *file,
   }
 }
 
+//! throws a nvrtcResultException if something went wrong
+#define NVRTC_SAFE_CALL_LOG(error, m_log)                                                 \
```

Comment:
hmm nicht ganz sicher ob mir gefällt eine Funktion zu schreiben nur für dieses eine mal, aber gut für die macros macht das wohl am meisten Sinn, kann man wohl schlecht in den Konstruktor von den Exceptions selbst verlagern...